client/src/components/RequestForm.js

```diff
@@ -1,83 +1,139 @@
+      price,
+      text,
+    };
+
+    alert(JSON.stringify(request));
+  }
 
-export default function ConcertCardBig() {
   return (
-    <Form type="submit">
+    <Form onSubmit={handleSubmit}>
       <Div>
-        <Label>Band/Künstler</Label>
-        <Input type="text" placeholder=""></Input>
+        <Input
+          required
+          value={artist}
+          onChange={(event) => {
+            setArtist(event.target.value);
+          }}
+        />
+        <FloatingLabel>Band/Künstler</FloatingLabel>
       </Div>
       <Div>
-        <Label>Datum</Label>
-        <Input type="date" placeholder=""></Input>
+        <Input
+          required
+          type="date"
+          value={date}
+          onChange={(event) => {
+            setDate(event.target.value);
+          }}
+        />
+        <FloatingLabel>Datum</FloatingLabel>
       </Div>
       <Div>
-        <Label>Location</Label>
-        <Input type="text" placeholder=""></Input>
+        <Input
+          required
+          value={location}
+          onChange={(event) => {
+            setLocation(event.target.value);
+          }}
+        />
+        <FloatingLabel>Location</FloatingLabel>
       </Div>
       <Div>
-        <Label>Stadt</Label>
-        <Input type="text" placeholder=""></Input>
+        <Input
+          required
+          value={city}
+          onChange={(event) => {
+            setCity(event.target.value);
+          }}
+        />
+        <FloatingLabel>Stadt</FloatingLabel>
       </Div>
       <Div>
-        <Label>Anfahrt</Label>
-        <Input type="text" placeholder=""></Input>
+        <Dropdown
+          required
+          value={arrival}
+          onChange={(event) => {
+            setArrival(event.target.value);
+          }}
+        >
+          <option value="">Wie willst du anreisen?</option>
+          <option value="auto">Auto</option>
+          <option value="öpnv">ÖPNV</option>
+          <option value="fahrrad">Fahrrad</option>
+          <option value="fuß">Zu Fuß</option>
+          <option value="mitfahrgelegenheit">Mitfahrgelegenheit</option>
+          <option value="sonstiges">Sonstiges</option>
+        </Dropdown>
       </Div>
       <Div>
-        <Label>Ticketpreis</Label>
-        <Input type="text" placeholder=""></Input>
+        <Input
+          required
+          type="number"
+          value={price}
+          onChange={(event) => {
+            setPrice(event.target.value);
+          }}
+        />
+        <FloatingLabel>Ticketpreis</FloatingLabel>
       </Div>
       <Div>
-        <Label>Dein Text</Label>
-        <Input type="text" placeholder=""></Input>
+        <Textarea
+          required
+          value={text}
+          onChange={(event) => {
+            setText(event.target.value);
+          }}
+        />
+        <FloatingLabel>Hier kommt dein Text rein</FloatingLabel>
       </Div>
+      <ButtonFull>Absenden</ButtonFull>
     </Form>
   );
 }
 
 const Div = styled.div`
```

Comment:
the naming here is a bit strange... maybe call it container?  and the only styling is the `position: relative`? 🤔

Reply:
I need this position property for the floating label :) And maybe I call the div "wrapper"?🤔

Reply:
yeah wrapper is a better idea 👍